src/wp-includes/default-filters.php

```diff
@@ -195,16 +195,17 @@
 add_filter( 'the_content', 'wpautop' );
 add_filter( 'the_content', 'shortcode_unautop' );
 add_filter( 'the_content', 'prepend_attachment' );
-add_filter( 'the_content', 'wp_filter_content_tags' );
```

Comment:
You have have seen this, but there is one instance of a plugin removing this filter at priority 10:

https://wpdirectory.net/search/01HBEBXA89CXBWDPZHN3C6FWNQ

Nevertheless, the code is commented out 😄 

```php
            /*
             * Temporarily unhook wp_filter_content_tags() since any tags
             * within the excerpt are stripped out. Modifying the tags here
             * is wasteful and can lead to bugs in the image counting logic.
             */
            //$filter_removed = remove_filter( 'the_content', 'wp_filter_content_tags' );
```

Reply:
In a GitHub search, there are ~13 non-core occurrences of this removal: https://github.com/search?q=%2Fremove_filter%5C%28%5Cs*%5B%27%22%5Dthe_content%5B%27%22%5D%2C%5Cs*%5B%27%22%5Dwp_filter_content_tags%5B%27%22%5D%2F&type=code&p=1

Reply:
So overall pretty safe to change the priority. I presume the dev note will call out the priority change.

Reply:
Also, I couldn't think of any specific reason to go with a priority higher than 12. As I found, there are scant removals of this filter in the plugin ecosystem, so if the priority is changed later it's not going to be the end of the world. 

I did do a search for plugins that are filtering `the_content` to do _something_ with images: https://wpdirectory.net/search/01HBHY8RKC5AARSB5J4M4FYJ3H

I didn't find any examples of a plugin adding an image with a priority above 12.

Reply:
All this to say, this will correctly fix the case I made a gist plugin for: https://gist.github.com/westonruter/faccfac310645d0c922f1c56d9ec1fbf